solargraph-standardrb.gemspec

```diff
@@ -18,6 +18,6 @@ Gem::Specification.new do |spec|
   spec.bindir = "exe"
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
-  spec.add_runtime_dependency "solargraph", "~> 0.39.1"
-  spec.add_runtime_dependency "standard", "~> 0.4.1"
+  spec.add_runtime_dependency "solargraph", ">= 0.39.1"
+  spec.add_runtime_dependency "standard", ">= 0.4.1", "< 1.0.0"
```

Comment:
@thewalkingtoast what about to remove `, "< 1.0.0"`. I guess it is compatible for greater 1.0 and when not, someone shouts very fast.